### PR TITLE
Initialize CFME with default value

### PR DESCRIPTION
**Pull request by machacekondra**
This patch add tasks to initialize CFME.

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @mgoldboi

File: defaults/main.yml [hunk 1]
```diff
@@ -24,6 +25,13 @@ miq_vm_disk_size: 66GiB
 miq_vm_disk_interface: virtio
 miq_vm_disk_format: cow
 
+# Additional disks
```
Comment (mgoldboi) [inline defaults/main.yml]: worth mentioning it's the DB disk

File: defaults/main.yml [hunk 2]
```diff
@@ -24,6 +25,13 @@ miq_vm_disk_size: 66GiB
...
+miq_vm_disks:
+  - name: miq_db_disk
+    size: 50GiB
+    interface: virtio
+    format: cow
```
Comment (mgoldboi) [inline defaults/main.yml]: please convert the DB disk format to RAW (performance)

File: tasks/init_cfme.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+- name: Add host alias of appliance
+  no_log: true
+  add_host:
+    hostname: "{{ miq_ip_addr }}"
+    ansible_host: "{{ miq_ip_addr }}"
+    ansible_password: "{{ miq_password }}"
+    ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+
+- name: Fetch info about appliance
```
Comment (mgoldboi) [inline tasks/init_cfme.yml]: patch validation fails for me here:
fatal: [10.12.50.53]: UNREACHABLE! => {
    "changed": false, 
    "msg": "Authentication failure.", 
    "unreachable": true
}
for some reason it's trying to connect to the engine

File: tasks/init_cfme.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      name: "{{ item.name }}"
+      vm_name: "{{ miq_vm_name }}"
+      interface: "{{ item.interface | default(omit) }}"
+      size: "{{ item.size | default(omit) }}"
+      format: "{{ item.format | default(omit) }}"
+      storage_domain: "{{ item.storage | default(miq_vm_disk_storage) | default(disk_storage_domain.name) }}"
+    with_items: "{{ miq_vm_disks }}"
+  
+  - name: Initialize CloudForms
+    command: "appliance_console_cli -i -r 0 -U root -p {{ miq_password }} -b /dev/vdb -k -f"
```
Comment (mgoldboi) [inline tasks/init_cfme.yml]: can we also pass the command line as parameter, so we can actually give the user the option to manipulate the installation procedure to be more modular.

**Review by mgoldboi — COMMENTED** [review]
please see inline comments

File: tasks/init_cfme.yml (hunk 3 above)
Comment (machacekondra) [reply tasks/init_cfme.yml]: It's trying to connect to '10.12.50.53', is that engine or CF?

Comment (machacekondra) [reply tasks/init_cfme.yml]: Most probably you've passed wrong `miq_password`

**Pull request merged**